### PR TITLE
[INTEL MKL] Fixed MKL QuantizeV2 operator.

### PR DESCRIPTION
**Pull request by mdfaijul**
There has been a new attribute added to QuantizeV2 (commit id: 4a8064512ca53c). Since MKL version was not updated accordingly, all quantization models with MKLDNN have been failing. This PR updates the MKL version to adapt those changes.

This PR also contains a small fix to `tensorflow/core/graph/mkl_layout_pass.cc` that was causing an abort in debug mode.

### CODE REVIEW

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR! I have one question.

File: tensorflow/core/kernels/mkl_quantize_op.cc [hunk 1]
```diff
@@ -55,16 +65,49 @@ template <typename Device, typename T>
 class MklQuantizeV2Op : public OpKernel {
  public:
   explicit MklQuantizeV2Op(OpKernelConstruction* ctx) : OpKernel(ctx) {
+    half_range_ =
+        !std::is_signed<T>::value
```
[the remark this answers is not in the document]
Comment (mdfaijul) [reply tensorflow/core/kernels/mkl_quantize_op.cc]: @penpornk Thanks for your comments. I have followed Eigen library based implementation https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/quantize_op.cc#L62-L67
We are actually not using this. Better, I would remove this variable and the related statement.

[the remark this answers is not in the document]
Comment (mdfaijul) [reply tensorflow/core/kernels/mkl_quantize_op.cc]: @penpornk Removed `half_range_` member variable.

**Review by penpornk — APPROVED** [review]
Sounds good. Thank you for the change!

**Pull request merged**